docs/404.html

```diff
@@ -3,9 +3,9 @@
   <head>
     <meta charset="utf-8">
 <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
-<meta name="generator" content="Hugo 0.78.2" />
+<meta name="generator" content="Hugo 0.80.0" />
 
-<META NAME="ROBOTS" CONTENT="NOINDEX, NOFOLLOW">
+<META NAME="ROBOTS" CONTENT="INDEX, FOLLOW">
```

Comment:
Any effect for this change? (`NOFOLLOW`=>`FOLLOW`)

Reply:
that was not a manual change, so i cannot explain it.

Reply:
All i did was run the commands i added to the description.

Possibly someone else ran it with different env vars defined.
https://github.com/kubernetes-sigs/cli-experimental/blob/master/site/themes/docsy/layouts/partials/head.html#L4